pkg/apis/testing/v1alpha1/testsuite_types.go

```diff
@@ -131,17 +137,18 @@ type TestResult struct {
 	// Test name
 	Name       string          `json:"name"`
 	Namespace  string          `json:"namespace"`
+	Status     TestStatus      `json:"status"`
 	Executions []TestExecution `json:"executions"`
 }
 
 // TestExecution provides status for given test execution
 type TestExecution struct {
-	ID             string              `json:"id"` // ID is equivalent to a testing Pod name
-	Status         TestExecutionStatus `json:"status"`
-	StartTime      *metav1.Time        `json:"startTime,inline,omitempty"`
-	CompletionTime *metav1.Time        `json:"completionTime,inline,omitempty"`
-	Reason         string              `json:"reason"`
-	Message        string              `json:"message"`
+	ID             string       `json:"id"` // ID is equivalent to a testing Pod name
```

Comment:
Having a comment like this makes it render it as a description to PodPhase rather than ID. Here's the fix.
```suggestion
	// ID is equivalent to a testing Pod name
	ID             string       `json:"id"` 
```